src/python/pants/engine/target.py

```diff
@@ -1162,83 +1162,10 @@ def compute_value(
 
 
 # -----------------------------------------------------------------------------------------------
-# Common Fields used across most targets
+# Sources and codegen
 # -----------------------------------------------------------------------------------------------
 
 
-class Tags(StringSequenceField):
```

Comment:
Ignore all the red the next few lines. This all was just moved lower in the file. No modifications.